CONTRIBUTING.md

```diff
@@ -62,13 +62,19 @@ juju model-config logging-config="<root>=INFO;unit=DEBUG"
 juju status
 juju debug-log
 ```
+### Configure juju
+```
+# customise update status hook interval:
+juju model-config update-status-hook-interval=1m
```

Comment:
Does this not effect every application in the model? Were there any alternatives in which we did not have to mess with the fundamental defaults?

Reply:
It does, though this model will be specific to superset and related charms, for which I would also use this `check`. An alternative would be we leave the default as is and it's not possible for the charm to be marked as `Active` for a minimum of 5 minutes. Documentation could be updated to say deploy superset and wait 5 minutes for it to become active. I have seen similar in other charms. 

I will leave like this for now but if this becomes an issue for other charms in the model I will consider requiring the 5 minute wait. Unless you have an alternative suggestion in mind?

Reply:
Maybe I do not fully understand what is going on here. That interval is a change to make the charm hooks more responsive, not a hard requirement, right? I have the feeling that charms should just work without messing with the underlying parameters. But that is just my opinion so if you think this is fine then go with it, I don't think it is a big problem.

Reply:
I've added this as a note to the `CONTRIBUTING.md` and mentioned it is optional.